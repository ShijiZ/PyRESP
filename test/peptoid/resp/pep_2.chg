All values are reported in atomic units
%FLAG TITLE
 molecule 1: peptoid

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
   2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
   3   7.1693530E+00   3.8731120E-01   1.6170360E-01
   4   5.4836410E+00   4.7919210E-01   3.0292610E+00
   5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
   6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
   7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
   8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
   9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
  10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
  11   1.5824870E+00   3.9493260E+00   1.3602580E+00
  12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
  13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
  14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
  15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
  16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
  17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
  18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
  19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
  20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
  21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
  22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          1         -1     6.5743130E-02
   2          6         -1    -1.7238758E-01
   3          1         -1     5.3875841E-02
   4          1         -1     7.9996452E-02
   5          6         -1     5.4713215E-01
   6          8         -1    -5.7436000E-01
   7          7         -1    -2.1139910E-01
   8          6          0    -2.2332560E-01
   9          1          0     1.0206188E-01
  10          1          9     1.0206188E-01
  11          1          9     1.0206188E-01
  12          6          0     3.6117913E-02
  13          1          0     3.1510573E-02
  14          1         13     3.1510573E-02
  15          6         -1     5.9730000E-01
  16          8         -1    -5.6790000E-01
  17          7         -1    -4.6437721E-01
  18          1         -1     2.9579457E-01
  19          6         -1    -2.5108284E-02
  20          1         -1     9.7916354E-02
  21          1         -1     3.9985850E-02
  22          1         -1     5.5788718E-02

