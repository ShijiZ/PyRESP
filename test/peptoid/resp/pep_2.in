 second stage of RESP. Peptoid for resp charge calculations
 &cntrl 
 nmol = 1,
 iqopt = 2,
 ihfree = 1,
 qwt = 0.001,
 ioutopt = 1,
 ipol = 0,
 ipermdip = 0,
 &end 
    1.0
peptoid
    0   22
    1   -1
    6   -1
    1   -1
    1   -1
    6   -1
    8   -1
    7   -1
    6    0
    1    0
    1    9
    1    9
    6    0
    1    0
    1   13
    6   -1
    8   -1
    7   -1
    1   -1
    6   -1
    1   -1
    1   -1
    1   -1




