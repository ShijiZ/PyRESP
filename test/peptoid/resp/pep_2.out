
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
  second stage of RESP. Peptoid for resp charge calculations
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 2
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00100000

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: peptoid
 Total charge (ich):  0
 Number of centers: 22
    1    1   -1
    2    6   -1
    3    1   -1
    4    1   -1
    5    6   -1
    6    8   -1
    7    7   -1
    8    6    0
    9    1    0
   10    1    9
   11    1    9
   12    6    0
   13    1    0
   14    1   13
   15    6   -1
   16    8   -1
   17    7   -1
   18    1   -1
   19    6   -1
   20    1   -1
   21    1   -1
   22    1   -1

 -----------------------------
 Atom Ivary Information
 -----------------------------
    1   -1
    6   -1
    1   -1
    1   -1
    6   -1
    8   -1
    7   -1
    6    0
    1    0
    1    9
    1    9
    6    0
    1    0
    1   13
    6   -1
    8   -1
    7   -1
    1   -1
    6   -1
    1   -1
    1   -1
    1   -1

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   22
 Weight factor on initial charge restraints=  0.001000
 There are  1 charge constraints

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    22
 total number of esp points =   983

 center       X               Y               Z
    1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
    2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
    3   7.1693530E+00   3.8731120E-01   1.6170360E-01
    4   5.4836410E+00   4.7919210E-01   3.0292610E+00
    5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
    6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
    7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
    8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
    9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
   10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
   11   1.5824870E+00   3.9493260E+00   1.3602580E+00
   12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
   13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
   14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
   15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
   16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
   17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
   18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
   19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
   20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
   21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
   22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00


 ------------------------
 Optimization Information
 ------------------------
 Since iqopt>1,   22 new q0 values will be read from file
 pep_1.chg

 Number of unique UNfrozen charge centers =     4

 Non-linear optimization requested.
  qchnge =    6.1729094480E-02
  qchnge =    2.8301714283E-03
  qchnge =    4.2410262252E-04
  qchnge =    7.3365959000E-05
  qchnge =    1.2998405989E-05
  qchnge =    2.3125318224E-06
  qchnge =    4.1172150591E-07

 Convergence in    6 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: peptoid

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         1        0.065743      0.065743     -1       0.000000
    2         6       -0.172388     -0.172388     -1       0.005018
    3         1        0.053876      0.053876     -1       0.000000
    4         1        0.079996      0.079996     -1       0.000000
    5         6        0.547132      0.547132     -1       0.001798
    6         8       -0.574360     -0.574360     -1       0.001715
    7         7       -0.211399     -0.211399     -1       0.004276
    8         6       -0.225495     -0.223326      0       0.004087
    9         1        0.092440      0.102062      0       0.000000
   10         1        0.104017      0.102062      9       0.000000
   11         1        0.121893      0.102062      9       0.000000
   12         6       -0.004004      0.036118      0       0.009405
   13         1        0.033708      0.031511      0       0.000000
   14         1        0.059439      0.031511     13       0.000000
   15         6        0.597300      0.597300     -1       0.001651
   16         8       -0.567900     -0.567900     -1       0.001734
   17         7       -0.464377     -0.464377     -1       0.002105
   18         1        0.295795      0.295795     -1       0.000000
   19         6       -0.025108     -0.025108     -1       0.009699
   20         1        0.097916      0.097916     -1       0.000000
   21         1        0.039986      0.039986     -1       0.000000
   22         1        0.055789      0.055789     -1       0.000000

 Sum over the calculated charges:     -0.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          4.4905295E-01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   3.4431644E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.8715530E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         8.7564909E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        3.87694   -1.11877    2.63385   -2.61569

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   10.96986
        Y    7.79840   -8.29938
        Z    5.77069   -1.75818   -2.67048

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   15.09399
        Y    0.00000   -2.59623
        Z    0.00000    0.00000  -12.49776